.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+
+# This is a generic workflow for releasing a Puppet module.
+# It requires that the caller sets `secrets: inherit` to ensure
+# that secrets are visible from steps in this workflow.
```

Comment:
can probably remove this comment as it's in the context of the reusable workflows